starboard/client_porting/icu_init/icu_init.cc

```diff
@@ -44,7 +44,8 @@ void Initialize() {
   data_path += kSbFileSepString;
   data_path += "icu";
 
-#if SB_IS(EVERGREEN)
+// TODO: Ask Kaido/YAvor if this is needed.
```

Comment:
Let's leave this one to be Evergreen. Also we should actually clean it up as we changed where we store the ICU tables. But that would be different change.